### ISSUE TITLE
About NUS-WIDE dataset

### ISSUE THREAD

**Issue by spoonnotfound** [kevinlin311tw/Caffe-DeepBinaryCode#28]
In your paper, "The downloaded images are divided into a training set of 97,214 images and a test set of 65,075 images". To know it for sure, can you upload your NUS-WIDE training and testing txt? Thank you!

**Comment by kevinlin311tw** [comment]
Thanks for your interest in this work. 

It is hard to get the original data since I have left the previous team two years ago. I checked the data for a while, and I found something might be relevant. However, I cannot guarantee these are the exact file lists we used two years ago.  

https://www.dropbox.com/s/ipwatbotjbu1nws/original-train-file-list.txt?dl=0
https://www.dropbox.com/s/un3a62g9lhhf200/original-test-file-list.txt?dl=0
